.github/workflows/pull_request.yaml

```diff
@@ -1,7 +1,7 @@
 name: Pull Request
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
Temporarily set to `pull_request` to be able to test the new version of golangci-lint in the PR workflow, need to change this back after approval and before merge if pipeline is successful